new-integration-tests/pom.xml

```diff
@@ -73,6 +75,12 @@
             <version>2.10.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+            <version>1.6.0</version>
```

Comment:
please use dependency management section for version numbers (like operator parent pom). Please fix the other version number in here while you are there (that slipped through reviews).